content/docs/cli/controller.md

```diff
@@ -20,6 +20,7 @@ Flags:
       --acme-http01-solver-resource-limits-memory string    Defines the resource limits Memory size when spawning new ACME HTTP01 challenge solver pods. (default "64Mi")
       --acme-http01-solver-resource-request-cpu string      Defines the resource request CPU size when spawning new ACME HTTP01 challenge solver pods. (default "10m")
       --acme-http01-solver-resource-request-memory string   Defines the resource request Memory size when spawning new ACME HTTP01 challenge solver pods. (default "64Mi")
+      --acme-http01-solver-run-as-non-root                  Defines the ability to run the http01 solver as root for troubleshooting issues (default true)
       --add_dir_header                                      If true, adds the file directory to the header of the log messages
```

Comment:
note: all changes like this (adding 'missing' things to API/CLI docs) are a result of running the generation scripts locally and committing the changes. There's no runtime effect, since the generation scripts are run at build-time, too - but it's nice to have the repo fully up to date.